components/function-controller/internal/controllers/serverless/function_controller.go

```diff
@@ -64,15 +64,34 @@ func (r *FunctionReconciler) calculateImageTag(instance *serverlessv1alpha1.Func
 }
 
 func (r *FunctionReconciler) updateStatus(ctx context.Context, result ctrl.Result, instance *serverlessv1alpha1.Function, condition serverlessv1alpha1.Condition) (ctrl.Result, error) {
+	return r.updateStatus2(ctx, result, instance, condition, nil)
```

Comment:
refactor will be applied in https://github.com/kyma-project/kyma/pull/9111/files